pgx_test/src/dune

```diff
@@ -1,3 +1,3 @@
 (library
  (name pgx_test)
- (libraries alcotest base64 pgx))
+ (libraries alcotest base64 pgx pgx_value_core))
```

Comment:
Why does Pgx_test need a dependency on this? I thought Pgx_value_core handled its own testing.

Reply:
this should be removed. I'll update

Reply:
This is now removed